client/shipping-label/views/purchase/steps/packages/package-info.js

```diff
@@ -81,21 +94,40 @@ const PackageInfo = ( { packageId, selected, all, flatRateGroups, unpacked, dime
 		} );
 
 		return (
-			<FormSelect onChange={ packageOptionChange } value={ pckg.box_id }>
-				{ _.map( groups, ( group, groupId ) => {
-					if ( _.isEmpty( group.definitions ) ) {
-						return null;
-					}
-
-					return <optgroup label={ group.title } key={ groupId }>
-						{ _.map( group.definitions, renderPackageOption ) }
-					</optgroup>;
-				} ) }
-			</FormSelect>
+			<div>
+				<div className="wcc-package-items-header">
+					<FormLegend className="wcc-package-item__name">{ __( 'Package of Multiple Items' ) }</FormLegend>
+				</div>
+				<FormSelect onChange={ packageOptionChange } value={ pckg.box_id } isError={ pckgErrors.box_id }>
+					<option value={ 'not_selected' } key={ 'not_selected' }>{ __( 'Please select a box' ) }</option> )
```

Comment:
This matches the design, but should probably be "package" or "package type", since envelopes are options. cc: @olaolusoga